### DISCUSSION TITLE
updates-to-the-stronglytypedid-library/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#402] [Announcements]
# updates-to-the-stronglytypedid-library/

In this post I describe some recent major updates to my StronglyTypedId NuGet package, describe why I made the changes, and demonstrate how they work

https://andrewlock.net/updates-to-the-stronglytypedid-library/

**Comment by espenrl** [comment]
Thank you so much for putting the effort into this new design. Now it´s even less verbose, who thought, and more extensible. I´ve been using StronglyTypedId a long time to safe guard myself from mixing up guids in my programs, it´s a genious tool.

**Comment by ProTip** [comment]
Composite keys via complex types in EF Core is going to be great.

**Comment by raulidavid** [comment]
could not be mapped because the database provider does not support this type. Consider converting the property value to a type supported by the database using a value converter 

.NET 8 with Strongly beta7 :(

**Comment by Lepollo3000** [comment]
I don´t know where to put the [assembly:StronglyTypedIdDefaults(Template.Int)] tag. It's not specific enough :C

**Reply by andrewlock** [reply]
You can put it literaly in any file, but an assembyinfo.cs file is most common 🙂
